src/lib/rmt/ssl/certificate_generator.rb

```diff
@@ -98,7 +98,7 @@ def generate(common_name, alt_names, ca_password)
       Yast::SCR.Write(Yast.path('.target.string'), @ssl_paths[:ca_config], config_generator.make_ca_config)
 
       RMT::Execute.on_target!(
-        'openssl', 'genrsa', '-aes256', '-passout', 'stdin', '-out', @ssl_paths[:ca_private_key], OPENSSL_KEY_BITS,
+        'openssl', 'genpkey', '-algorithm RSA', '-aes256 -out', @ssl_paths[:ca_private_key], '-pkeyopt rsa_keygen_bits:', OPENSSL_KEY_BITS,
```

Comment:
Is there a reason to drop `-passout stdin`? I would keep it since standards might change and then this breaks.

Reply:
`genpkey` does not accept those flags. It does prompt from stdin by default